src/splitWhen.js

```diff
@@ -0,0 +1,23 @@
+var _curry2 = require('./internal/_curry2');
+var complement = require('./complement');
+var dropWhile = require('./dropWhile');
+var takeWhile = require('./takeWhile');
+
+
+/**
+ * Splits a given list at the first occurance of a satisfied predicate.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig (a -> Boolean) -> [a] -> [[a], [a]]
+ * @param {Function} pred The predicate that determines where the array is split.
+ * @param {Array} list The array to be split.
+ * @return {Array}
+ * @example
+ *
+ *      R.splitWhen(R.equals(2), [1, 2, 3, 1, 2, 3]);   //=> [[1], [2, 3, 1, 2, 3]]
+ */
+module.exports = _curry2(function splitWhen(pred, list) {
+  return [takeWhile(complement(pred), list), dropWhile(complement(pred), list)];
+});
```

Comment:
This looks good to me.

Sorry we dropped the ball earlier.  I've been mostly disengaged for a few weeks, and I think others mostly have too.  I probably will be so for a few more, but I will try to watch what's happening.